tutorial/app_specific_data.js

```diff
@@ -0,0 +1,48 @@
+/* global Web3 globalState render */
+
+// Setup
+
+const Connect = window.uportconnect.Connect
+const appName = 'AppDataTutorial'
+const connect = new Connect(appName)
+const web3 = connect.getWeb3()
+const UportRegistry = window.uportregistry.contractData
+console.log(UportRegistry)
+const appId = '0xe2fef711a5988fbe84b806d4817197f033dde050'
+
+const Registry = web3.eth.contract(UportRegistry.abi)
+const reg = Registry.at(UportRegistry.networks['3'].address)
+
+// uPort connect
+
+const uportConnect = () => {
+  connect.requestCredentials().then((credentials) => {
+    console.log(credentials)
+    globalState.uportId = credentials.address
```

Comment:
Once we support MNID's this could be an MNID and not a hex address. So you will need to check for it and extract it. 

@zachferland we should think up a more permanent solution for this.